src/collectd_dogstatsd.py

```diff
@@ -106,13 +106,12 @@ def dims_from_tags(tags, metric):
         return ret
     for tag in tags:
         parts = tag.split(":", 1)
-        if len(parts) == 0:
-            # Skip labels
-            continue
-        if len(parts) == 1:
-            collectd.error(
-                "skipping malformed dogstatsd tag: '{0}' for metric: '{1}'"
+        # Skip labels and malformed tags
+        if len(parts) < 2:
```

Comment:
When could `len(parts)` ever be anything except for `1`?

Reply:
I don't think it ever could be.  I think split should always return a list with at least 1 element. Whoever originally wrote this was looking for a length of 0.